server.js

```diff
@@ -0,0 +1,14 @@
+const PORT = 3000;
+const express = require("express");
+const bodyParser = require("body-parser");
```

Comment:
Good work on your code! Looking at your code, the body-parser doesn't look like it's quite necessary here.